catalog.html

```diff
@@ -1,264 +1,315 @@
+                  <input class="control-input" type="checkbox" name="Парковка" checked>
+                  Парковка
+                </label>
+              </li>
+
+              <li class="catalog-filter-item">
+                <label class="control">
+                  <input class="control-input" type="checkbox" name="Wi-Fi">
+                  Wi-Fi
+                </label>
+              </li>
+            </ul>
+          </fieldset>
+
+          <fieldset class="catalog-filter-group">
+            <legend class="catalog-filter-title">Тип Жилья:</legend>
+            <ul class="catalog-filter-list">
+
+
+            <li class="catalog-filter-item">
+                <label class="control">
+                  <input class="control-input" type="radio" name="Гостиница">
+                  Гостиница
+              </label>
+              </li>
+
+            <li class="catalog-filter-item">
+                <label class="control">
+                  <input class="control-input" type="radio" name="Мотель">
+                  Мотель
+                </label>
+              </li>
+
+            <li class="catalog-filter-item">
+                <label class="control">
+                  <input class="control-input" type="radio" name="Апартаменты">
+                  Апартаменты
+              </label>
+               </li>
+              </ul>
+          </fieldset>
+          <fieldset>
+            <legend>Стоимость, <span>₽</span></legend>
+            <label>от<input type="text" value="0"></label>
+            <label>до<input type="text" value="1000"></label>
+          </fieldset>
+
+          <button type="submit" name="popup-button">Применить</button>
+          <button type="reset" name="popup-button">Сбросить</button>
+        </form>
+      </section>
+
+
+        <h2 class="page-footer-title">Найдено гостиниц:38</h2>
+        <select class="select-control" aria-label="Сортировка">
+          <option value="popular">Сначала популярное</option>
+          <option value="cheap">Сначала дешёвое</option>
+          <option value="expensive">Сначала дорогое</option>
+        </select>
+
+  <a href="#" class="button button-light">
+    <span class="visually-hidden">Показать список</span>
+ </a>
+
+  <a class="button" href="#">
+    <span class="visually-hidden">Показать карточки</span>
+  </a>
+
+  <a class="button" href="#">
+    <span class="visually-hidden">Показать таблицей</span>
+  </a>
+
+  <div class="catalog-soft-in-check">
+    <a href="#" class="sort_btn block"><img src="images/icon1.svg" width="16" height="14" alt="Блоками"></a>
+    <a href="#" class="sort_btn current"><img src="images/icon2.svg" width="16" height="14" alt="Экраном"></a>
+    <a href="#" class="sort_btn table"><img src="images/icon3.svg" width="16" height="14" alt="Таблицей"></a>
+  </div>
+
+
+
+
+  <div class="product-card">
+    <a class="product-card-link" href="product.html">
+ <img class="card-image" src="images/hotel amara.jpg" width="300" height="212"
```

Comment:
```suggestion
 <img class="card-image" src="images/hotel-amara.jpg" width="300" height="212"
```
Выше писал, что в названии файлов пробела не должно быть. `hotel amara.jpg` можно исправить на `hotel-amara.jpg`